src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -274,15 +274,11 @@ class Accounts extends Component {
         );
         this.resize();
 
-        if ((this.props.requestStatus.instructor[GET][REQUEST_ALL] !== 200 ||
+        const isLoading = (this.props.requestStatus.instructor[GET][REQUEST_ALL] !== 200 ||
             this.props.requestStatus.student[GET][REQUEST_ALL] !== 200 ||
-            this.props.requestStatus.parent[GET][REQUEST_ALL] !== 200)
-        ) {
-            return (<Loading />)
-        }
+            this.props.requestStatus.parent[GET][REQUEST_ALL] !== 200);
```

Comment:
You can utilize `isLoading` in `hooks.js` (not an actual hook):

   const isLoading = hooks.isLoading(instructorStatus, studentStatus, parentStatus);